src/test/java/com/hotels/beeju/extensions/ThriftHiveMetaStoreJUnitExtensionTest.java

```diff
@@ -95,4 +95,11 @@ public void createDatabaseNullName() {
   public void createDatabaseInvalidName() {
     assertThrows(InvalidObjectException.class, () -> hiveDefaultName.createDatabase(""));
   }
+  
+  @Test
+  public void thriftPort() {
+    int thriftPort = 3333;
```

Comment:
Just a random port for the sake of testing?

Reply:
yes.